### PR TITLE
Allow the user to set the elasticmq version

### PR DESCRIPTION
**Pull request by C4n4rd0**

### CODE REVIEW

File: serverless-offline-elasticmq/types/elasticMQ.d.ts [hunk 1]
```diff
@@ -14,6 +14,7 @@ export interface ElasticMQLaunchOptions {
 
 
 export interface ElasticMQConfig {
+  version: string;
```
Comment (YOU54F) [inline serverless-offline-elasticmq/types/elasticMQ.d.ts]: should we make the version optional and default to the latest available at time of merging. that way people aren't forced to set the version themselves, if they don't want to?

File: serverless-offline-elasticmq/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "serverless-offline-elasticmq",
-  "version": "0.0.11",
+  "version": "0.0.12",
```
Comment (YOU54F) [inline serverless-offline-elasticmq/package.json:3]: If we make the option required, so users need to now add in the version, we will need to bump this as at least a minor

**Comment by YOU54F** [comment]
changes look good, just a suggestion on making the version optional, the the version number dependant on which option we take. 

Will look to get this tested locally and merged in and released early next week. I'll also do a dependency update as well, as it has been a while

File: serverless-offline-elasticmq/types/elasticMQ.d.ts (hunk 1 above)
Comment (C4n4rd0) [reply serverless-offline-elasticmq/types/elasticMQ.d.ts]: That's a question I asked myself and honestly, I don't know which option is the best. My first thought was that as there's a breaking change in the AWS SDK API, maybe it's best to force the user to set the version. But as second thought, I guess it's easier to make it optional. So let's do that. I've updated the code to make this field optional.

**Review by YOU54F — APPROVED** [review]

**Pull request merged**